hooks/checkout

```diff
@@ -254,8 +254,13 @@ main() {
     initialize_local_repo
 
     # set origin in case the repo was changed by the pre-checkout hook
-    log_info "Setting origin to ${BUILDKITE_REPO}"
-    git remote set-url origin "${BUILDKITE_REPO}"
+    if [[ -v GITHUB_FETCH_APP_ACCESS_TOKEN ]]; then
+      log_info "Setting origin to [[masked]]"
```

Comment:
can we print the repo here, just without the token? to avoid any confusion to what repo we are cloning from

Reply:
done